README.md

```diff
@@ -0,0 +1,35 @@
+# Benchy
+
+Hello, I am an electric benchy, I can't go very fast or very far and if you drive me people will think you are awesome.
```

Comment:
? GPT?

Reply:
ahh, the electric car, yes, sorry, I don't get references like these out of context :D